### ISSUE TITLE
Add Facebook and LinkedIn to company details default fields

### ISSUE THREAD

**Issue by richardmtl** [Automattic/WP-Job-Manager#1821]
Requested here: https://wordpress.org/support/topic/feature-update-request/

Yes, we can add fields, but I suggest that maybe they can become part of the default group of fields that ship with the plugin.

**Comment by Hastibe** [comment]
Yes, this would be very helpful, and is a feature that also has been requested by the employers who use our job board.

**Comment by zatakk** [comment]
Not so sure of fb, but Linkedin is a definite MUST !! Please give the option to feature linkedin profile on the candidate dashboard.

**Comment by jinnypark** [comment]
Requested here: 4534735-zd-woothemes
